java/lib/src/main/java/truelayer/signing/Verifier.java

```diff
@@ -105,6 +105,23 @@ public Verifier body(String body) {
         return this;
     }
 
+    /**
+     * Add a Map of headers.
+     * Warning: Only a single value per header name is supported.
+     *
+     * @param headers  - must not be null
+     * @return the Verifier instance
+     * @throws IllegalArgumentException if the provided params are null
+     */
+    public Verifier headers(Map<String, String> headers) {
```

Comment:
`Map` makes sense to have :+1: Are there other obvious std-lib types that http libraries would use for headers? We could add overloads if so...

Reply:
we can add later if needed